packages/codec/lib/format/values.ts

```diff
@@ -439,6 +449,9 @@ export interface FunctionExternalValue {
   type: Types.FunctionExternalType;
   kind: "value";
   value: FunctionExternalValueInfo;
+  interpretations: {
+    contractEnsName?: StringValueInfo;
```

Comment:
It's weird that this is an interpretation here. I guess it's too much trouble to move this closer to where the address field appears?

Reply:
Yeah I mean where else would it go?  I'm not looking to break things too hard here.

Reply:
Yeah, ok, that's reasonable.